### PR TITLE
doc(README): update chainloader and add sys loading

### PR DESCRIPTION
**Pull request by AtomToast**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,16 +5,32 @@ tutorial: https://github.com/s-matyukevich/raspberry-pi-os.git
...
\ No newline at end of file
+- ![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)![#f02015](https://via.placeholder.com/15/f02015/000000?text=+)
+  ---> Chainloader is waiting for kernel
+
+## Loading the system
+
+Once the Chainloader is waiting for the kernel size (![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)![#f02015](https://via.placeholder.com/15/444444/000000?text=+)) you can build and load the image onto the Raspberry Pi with `make send`.
+
+
+If you already loaded and booted the system you can restart the Pi, to start the chainloader again and rebuild + send the new image, with `make resend`.
```
Comment (anarchuser) [inline README.md:36]: It may be important to add that `make resend` will freeze the raspberry if the raspberry was waiting for `make send`

**Review by anarchuser — DISMISSED** [review]
looks good, I only noted two things that you can look into, but they are optional

File: README.md [hunk 2]
```diff
@@ -5,16 +5,32 @@ tutorial: https://github.com/s-matyukevich/raspberry-pi-os.git
...
-configuration). The chainloader uses the STATUS led ![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)
-and the POWER led ![#f02015](https://via.placeholder.com/15/f02015/000000?text=+) to indicate its current status. Read
-as follows:
+To make mounting of the SD card in the makefile work it is currently required to have set up the SD card in `/etc/fstab` to mount in the mnt directory at the project directory root level.  
+An example config line would be:
 
-* ![#444444](https://via.placeholder.com/15/444444/000000?text=+)![#444444](https://via.placeholder.com/15/444444/000000?text=+)
+```
+# /dev/mmcblk0p1
+UUID=7616-4FD8    /home/kjell/git/rose/mnt    vfat   auto,nofail,noatime,user,rw    0 2
```
[the remark this answers is not in the document]
Comment (AtomToast) [reply README.md:13]: Agreed, though I would like to eventually have it set up so the Make flash command can handle the sd card being mounted already. That isn't in the scope of this pr though

File: README.md (hunk 1 above)
Comment (AtomToast) [reply README.md:36]: Oh, yeah. Also should add that you have to hard reset and `make send` again if the pi freezes

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (anarchuser) [reply README.md:13]: yes, I looked a bit into it. There is a `remount` functionality, but it only allows changing attributes, not the mount point.
So we'd need to write a function that unomunts the sd card if it's mounted somewhere else, and then mounts the sd card if it's not mounted

File: README.md (hunk 1 above)
Comment (anarchuser) [reply README.md:36]: Technically, we could have the chainloader filter out the special character we use to reboot the raspberry.
We could just always send one character before we send the actual size. Wouldn't actually be hard

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (AtomToast) [reply README.md:13]: maybe we should fail if it is mounted already somewhere else but it should be fine if it is mounted correctly already? Just so we don't accidentally mess someones stuff up

[the remark this answers is not in the document]
Comment (AtomToast) [reply README.md]: implemented in #50

[the remark this answers is not in the document]
Comment (AtomToast) [reply README.md]: and the original comment got fixed in ca3355a

File: README.md (hunk 1 above)
Comment (AtomToast) [reply README.md:35]: this would need to be handled in a separate PR

**Review by anarchuser — APPROVED** [review]

**Pull request merged**